rbac/management/group/view.py

```diff
@@ -482,14 +483,26 @@ def roles(self, request, uuid=None):
 
         return Response(status=status.HTTP_200_OK, data=response_data.data)
 
+    def filtered_roles(self, roles, request):
+        """Return filtered roles for group from query params."""
+        role_filters = {}
+        for param_name, param_value in request.query_params.items():
+            if param_name in VALID_GROUP_ROLE_FILTERS:
+                attr_filter_name = param_name.replace('role_', '')
+                role_filters[f'{attr_filter_name}__icontains'] = param_value
```

Comment:
So we want to support just case insensitive search (Tom Jerry when we input tom), not returning e.g. Tommy when we input tom

Reply:
Correct, the other filters we have setup are case insensitive, so I implemented this one the same.